apps/web/src/components/layouts/AppLayout/Header/index.tsx

```diff
@@ -80,30 +86,65 @@ export type AppHeaderProps = {
   navigationLinks: INavigationLink[]
   currentUser: SessionUser | undefined
   breadcrumbs?: IBreadCrumb[]
+  sectionLinks?: INavigationLink[]
 }
 export default function AppHeader({
   breadcrumbs = [],
   navigationLinks,
   currentUser,
+  sectionLinks,
 }: AppHeaderProps) {
   const info = currentUser ? getUserInfoFromSession(currentUser) : null
   return (
-    <header className='sticky top-0 px-6 py-3 flex flex-row items-center justify-between border-b border-b-border'>
-      <Breadcrump breadcrumbs={breadcrumbs} />
-      <div className='flex flex-row items-center gap-x-6'>
-        <nav className='flex flex-row gap-x-2'>
-          {navigationLinks.map((link, idx) => (
-            <NavLink key={idx} {...link} />
-          ))}
-        </nav>
-        {info ? (
-          <Avatar
-            alt={info.name}
-            fallback={info.fallback}
-            className='w-6 h-6'
-          />
-        ) : null}
+    <header className='sticky top-0 flex flex-col border-b border-b-border'>
+      <div className='px-6 py-3 flex flex-row items-center justify-between '>
+        <Breadcrump breadcrumbs={breadcrumbs} />
+        <div className='flex flex-row items-center gap-x-6'>
+          <nav className='flex flex-row gap-x-2'>
+            {navigationLinks.map((link, idx) => (
+              <NavLink key={idx} {...link} />
+            ))}
+          </nav>
+          {info ? (
+            <Avatar
+              alt={info.name}
+              fallback={info.fallback}
+              className='w-6 h-6'
+            />
+          ) : null}
+        </div>
       </div>
+      {!!sectionLinks?.length && (
+        <div className='flex flex-row px-6 bg-background'>
+          {sectionLinks.map((link, idx) => (
+            <SectionLink key={idx} link={link} />
+          ))}
+        </div>
+      )}
     </header>
   )
 }
+
+const SectionLink = ({ link }: { link: INavigationLink }) => {
+  const rootPath = usePathname().split('/')[1]
+  const selected = rootPath === link.href!.split('/')[1]
+  const classes = classNames('px-4 py-2', {
+    'border-b-2 border-accent-foreground': selected,
+  })
+
+  return (
+    <div className={classes}>
+      <Link
+        href={
+          new URL(
+            `${window.location.protocol}//${window.location.host}${link.href!}`,
```

Comment:
Why this?

Reply:
next's LInk expects an instance of URL which needs the full url

Reply:
![image](https://github.com/user-attachments/assets/b914c6b6-2a5c-4c04-ae60-fab0f5666636)

Looks like it [accepts an string](https://nextjs.org/docs/app/api-reference/components/link)